src/nodes/Image.tsx

```diff
@@ -113,19 +126,29 @@ export default class Image extends Node {
           tag: "div[class=image]",
```

Comment:
```suggestion
          tag: "div[class~=image]",
```

Now that the image might have those additional classes applied this selector needs updating otherwise copy/paste between documents won't work.